config/initializers/business_time.rb

```diff
@@ -1,3 +1,3 @@
-Holidays.between(Date.today, 5.years.from_now, :us, :observed).map do
+Holidays.between(Date.today, 5.years.from_now, :us, :observed).each do
```

Comment:
I don't like using map when the goal is just iteration and not creating a new array